lib/foreman_maintain/cli/base.rb

```diff
@@ -98,6 +98,8 @@ def self.tags_option
       end
 
       def self.interactive_option
+        delete_duplicate_assumeyes_if_any
+
         option ['-y', '--assumeyes'], :flag,
```

Comment:
I think we can check here if options already have `--assumeyes` flag and opt to add accordingly.

Reply:
Previously, I have made same changes as you mentioned. 
But description of --assumeyes option is different in package update/install  procedure.
To make the consistency between interactive options and their descriptions throughout procedures I did modifications.